# cpu usage percentage 0 - 100.0
float32 usage

# number of cpu physical cores
uint8 physical_cores

# number of cpu physical and virtual cores
uint8 cores

# CPU min, max and current frequency expressed in Mhz.
# If min and max cannot be determined they are set to 0
float32 min_freq
float32 max_freq
float32 current_freq
